rdagent/scenarios/data_science/dev/runner/eval.py

```diff
@@ -54,6 +54,9 @@ def evaluate(
 
         # execute workflow
         stdout, execute_ret_code = implementation.execute_ret_code(env=env, entry="python -m coverage run main.py")
+        running_time = re.search(r"Total running time: (\d+\.\d+) seconds.", stdout).group(1)
```

Comment:
maybe be we can create a result class to make it more reasonable and extensible.
```
class Resuult:
     stdout:
     ret_code:
     execute_time: 
```